static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -198,6 +205,7 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       if (this.isMobile()) {
         this.updateCssForMobile();
       } else {
+        this.core.storage.set('DISABLE_RENDER_RESULTS', false);
```

Comment:
since the function `setupCssForBreakpoints` no longer just updates the CSS, what are your thoughts on renaming it to `setupMobileBreakpointListener`?

Reply:
sounds good!